docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -0,0 +1,72 @@
+# Signalen and the Standard for Public Code
+
+## [Code in the open](https://standard.publiccode.net/criteria/code-in-the-open.html)
+
+* Current backlog and bugracking is not public. Some issues referenced in code and commits are not easy to locate, like "SIG-1525". Transition to public bug-tracker?
+* Can the classification system be completely open sourced? How can we address risks and possibilities regarding (of theoretical) leaking of information. Maybe add anonymized MOR datasets or redo the classification algorithm with public available MOR data sets of which there are plenty. Can a basic Commonground version be made?
+* Citizens are not authenticated. Data is not cleaned up for sharing (privacy purposes). https://github.com/Amsterdam/signals/blob/master/docs/api-design.md#publiek-en-ongeauthenticeerd
+
+## [Bundle Policy And Source Code](https://standard.publiccode.net/criteria/bundle-policy-and-code.html)
+
+* Making the state machine configurable would enable Signalen to adapt to policy changes within Amsterdam and also enable replicators to customise Signalen for their workflows.
+
+## [Create reusable and portable code](https://standard.publiccode.net/criteria/reusable-and-portable-codebases.html)
+
+* There are several dependencies on Amsterdam services. Replicators need this to be configurable.
+* Technical roadmap is not published.
+* [Workflow](https://github.com/Amsterdam/signals/blob/master/api/app/signals/apps/signals/workflow.py) is currently hardcoded. Names of key states can be found elsewhere, need abstraction plan.
+* Email messages are [hardcoded](https://github.com/Amsterdam/signals/blob/master/api/app/signals/apps/email_integrations/core/messages.py).
+* Separate UI text from API model classes: [example](signals/api/app/signals/apps/signals/models/history.py).
+* Extended category questions currently live in front-end code, extract and make plugable.
+* Rename or alias "sia" prefixes to something less AMS specific (tiny nitpick).
+
+
+## [Welcome contributors](https://standard.publiccode.net/criteria/open-to-contributions.html)
+
+* Improve [CONTRIBUTING.md](https://github.com/Amsterdam/signals/blob/master/docs/CONTRIBUTING.md), perhaps with content from Amsterdam's [CONTRIBUTING.md](https://github.com/Amsterdam/amsterdam.github.io/blob/master/CONTRIBUTING.md)
+
+## [Maintain version control](https://standard.publiccode.net/criteria/version-control-and-history.html)
+
+* Good.
+    * Little note: not all commits state what they do or why, some just reference an issue, or have "[WIP](https://github.com/Amsterdam/signals/commit/2f9e2f73ee5dc4cdf67d6854d1a7361f6e6aaf9b)" commit messages.
```

Comment:
Will providing a template for commit messages help?

Reply:
Maybe! One of the projects I contribute to has a template and my experience is that it does encourage me to write better commit messages. Thus, yes, I'd say give it a try and see what the team thinks after a while.